src/main/resources/schema.graphqls

```diff
@@ -0,0 +1,45 @@
+type Donor {
+    sample: Sample
+}
+
+type Sample {
+    sampleType: String
+    sequencingStrategy: String
+    analysis: [Analysis]
+}
+
+type Analysis {
+    id: String!
+    type: String!
+    runs: [Run]
+}
+
+type Run {
+    runName: String!
+    workDir: String!
+    state: String
+    request: RunRequest
+}
+
+scalar JSON
+type RunRequest {
+    workflowUrl: String!
+    workflowParams: JSON!
+    workflowEngineParams: WorkflowEngineParams
+    workflowType: String
+    workflowTypeVersion: String
+}
+
+type WorkflowEngineParams {
+    revision: String
+    resume: String
+    workDir: String
```

Comment:
will be adding a field in here called `latest` which is a boolean, up to you if you want to include this now or wait